src/components/BatchAllocations/BatchAllocationLogs/ConfirmDeleteLogModal.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { ConfirmationModal } from '@folio/stripes/components';
+
+export const ConfirmDeleteLogModal = ({
+  onCancel,
+  onConfirm,
+  open,
+}) => {
+  return (
+    <ConfirmationModal
+      id="delete-batch-allocation-log"
+      open={open}
+      heading={<FormattedMessage id="ui-finance.allocation.batch.logs.modal.delete.title" />}
+      message={<FormattedMessage id="ui-finance.allocation.batch.logs.modal.delete.description" />}
+      onConfirm={onConfirm}
+      onCancel={onCancel}
+      cancelLabel={<FormattedMessage id="ui-finance.cancel" />}
```

Comment:
Probably you can omit that props and the modal will use translation from `stripes`